app/models.py

```diff
@@ -892,6 +892,71 @@ class BriefUser(db.Model):
     user_id = db.Column(db.Integer, db.ForeignKey('users.id'), primary_key=True)
 
 
+class BriefResponse(db.Model):
+    __tablename__ = 'brief_responses'
+
+    id = db.Column(db.Integer, primary_key=True)
+    data = db.Column(JSON)
+
+    brief_id = db.Column(db.Integer, db.ForeignKey('briefs.id'), nullable=False)
+    supplier_id = db.Column(db.Integer, db.ForeignKey('suppliers.supplier_id'), nullable=False)
+
+    created_at = db.Column(db.DateTime, index=True, nullable=False, default=datetime.utcnow)
+
+    brief = db.relationship('Brief')
+    supplier = db.relationship('Supplier', lazy='joined')
+
+    @validates('data')
+    def validates_data(self, key, data):
+        data = drop_foreign_fields(data, [
+            'supplierId', 'briefId',
+        ])
+        data = strip_whitespace_from_data(data)
+        data = purge_nulls_from_data(data)
+
+        return data
+
+    def validate(self, enforce_required=True, required_fields=None):
+        errs = get_validation_errors(
+            'brief-responses-{}-{}'.format(self.brief.framework.slug, self.brief.lot.slug),
```

Comment:
One to check with Ralph/Sarah I think - at the moment I think Rob is right (see, e.g. http://dm-prototype.herokuapp.com/page/respond/template-brief?key=outcomes.0), but not sure if this might change in future.